src/SFA.DAS.EmployerIncentives.Domain.UnitTests/ApprenticeshipIncentive/WhenCreatePayment.cs

```diff
@@ -54,8 +57,38 @@ public void Then_the_payment_is_not_created_when_the_pending_payment_is_not_vali
             _sut.Payments.Count.Should().Be(0);
         }
 
+        [TestCase("2000-07-30", "2020-08-01", ApprenticeshipEmployerType.Levy, SubnominalCode.Levy19Plus)]
+        [TestCase("2000-07-30", "2020-08-01", ApprenticeshipEmployerType.NonLevy, SubnominalCode.NonLevy19Plus)]
+        [TestCase("2002-08-30", "2020-08-01", ApprenticeshipEmployerType.Levy, SubnominalCode.Levy16To18)]
+        [TestCase("2002-08-30", "2020-08-01", ApprenticeshipEmployerType.NonLevy, SubnominalCode.NonLevy16To18)]
+        [TestCase("2001-08-02", "2020-08-01", ApprenticeshipEmployerType.NonLevy, SubnominalCode.NonLevy16To18)]
+        [TestCase("2001-08-01", "2020-08-01", ApprenticeshipEmployerType.NonLevy, SubnominalCode.NonLevy19Plus)]
+        public void Then_the_payment_is_created(DateTime dob, DateTime plannedStartDate, ApprenticeshipEmployerType employerType, SubnominalCode expectedSubnominalCode)
+        {
+            // arrange
+            var apprenticeship = new Apprenticeship(_fixture.Create<long>(), _fixture.Create<string>(),
+                _fixture.Create<string>(), dob, _fixture.Create<long>(), employerType);
+
+            _sutModel = _fixture.Build<ApprenticeshipIncentiveModel>()
+                .With(x=>x.PlannedStartDate, plannedStartDate)
+                .With(x=>x.Apprenticeship, apprenticeship)
+                .With(x => x.PaymentModels, new List<PaymentModel>()).Create();
+
+            _sut = Sut(_sutModel);
+
+            var pendingPayment = _sut.PendingPayments.First();
+
+            // act
+            _sut.CreatePayment(pendingPayment.Id, _collectionYear, _collectionMonth);
+
+            // assert
+            _sut.Payments.Count.Should().Be(1);
+            var actualPayment = _sut.Payments.First();
+            actualPayment.SubnominalCode.Should().Be(expectedSubnominalCode);
+        }
+
         [Test]
-        public void Then_the_payment_is_created()
+        public void Then_the_payment_is_created_with_correct_SubnominalCode()
```

Comment:
The test name implies a check for subnominal code, but this is done in another test